keg_auth/model/__init__.py

```diff
@@ -489,6 +497,40 @@ def _query_permission_mapping(cls):
         return sa.union(direct, via_bundle)
 
 
+class AttemptMixin(object):
+    # Form input data, e.g. username
+    user_input = sa.Column(sa.Unicode(512), nullable=False)
```

Comment:
After removing the link to user table, I decided not to include a `user_id` field on the attempt model. The main reason is to avoid making assumptions about the type of `user_id`. For example, some project may use UUIDs, in which case the `user_id` field in the attempt model would need to be overridden. Since it's not needed and for the sake of simplicity I just left it out.